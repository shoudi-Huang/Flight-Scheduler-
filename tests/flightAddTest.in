flights
location add Mumbai 19.08 72.741 0.4
location add NewDelhi 28.527 77.0688988 -0.123
locations
flight add
flight add monday 10:00 mumbai newdelhi
flight add monday 10:05 mumbai newdelhi 80 ###
flights
flight add mondate 09:00 mumbai newdelhi 150
flight add Tuesday 24:00 mumbai newdelhi 140
flight add Thursday -1:00 mumbai newdelhi 140
flight add WEDNESDAY 18:60 mumbai newdelhi 140
flight add FrIdAY 18:-50 mumbai newdelhi 150
flight add Saturday 1905 mumbai newdelhi 150
FLIGHT ADD sunday xx:01 mumbai newdelhi 150
FlIghT aDd sunday 21:yi1 mumbai newdelhi 150
Flight add Sunday 00:00 Sydney newdelhi 10
Flight add Monday 01:59 mumbai London 40
Flight add Monday 01:59 mumbai 1 40
flight add monday 18:00 mumbai newdelhi -1
flight add monday 18:00 mumbai newdelhi 0
flight add monday 18:00 mumbai newdelhi capacit1
flight add monday 18:00 mumbai mumbai 90
flight 0
location add Sydney -33.847927 150.651786 0.4
flight add monday 10:00 mumbai newdelhi 200
flight add monday 09:10 mumbai newdelhi 200
flight add monday 11:04 mumbai newdelhi 200
flight add monday 12:00 newdelhi mumbai 120
flight add Sunday 21:50 Sydney newdelhi 90
flight adD mOnDAy 12:40 neWdeLhI MuMbAi 120 ###
flight add Sunday 21:50 Sydney newdelhi 90
flight 1
flight add Sunday 23:50 Sydney newdelhi 90 ###
flight add Sunday 23:25 Sydney newdelhi 90
flight add Sunday 00:45 Sydney newdelhi 70 ###
flight add Saturday 23:46 Sydney newdelhi 70
flight add Sunday 00:10 Sydney newdelhi 70
flights
exit
