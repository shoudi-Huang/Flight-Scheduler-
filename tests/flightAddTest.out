User: Flights
-------------------------------------------------------
ID   Departure   Arrival     Source --> Destination
-------------------------------------------------------
(None)

User: Successfully added location Mumbai.

User: Successfully added location NewDelhi.

User: Locations (2):
Mumbai, NewDelhi

User: Usage:   FLIGHT ADD <departure time> <from> <to> <capacity>
Example: FLIGHT ADD Monday 18:00 Sydney Melbourne 120

User: Usage:   FLIGHT ADD <departure time> <from> <to> <capacity>
Example: FLIGHT ADD Monday 18:00 Sydney Melbourne 120

User: Successfully added Flight 0.

User: Flights
-------------------------------------------------------
ID   Departure   Arrival     Source --> Destination
-------------------------------------------------------
   0 Mon 10:05   Mon 11:40   Mumbai --> NewDelhi

User: Invalid departure time. Use the format <day_of_week> <hour:minute>, with 24h time.

User: Invalid departure time. Use the format <day_of_week> <hour:minute>, with 24h time.

User: Invalid departure time. Use the format <day_of_week> <hour:minute>, with 24h time.

User: Invalid departure time. Use the format <day_of_week> <hour:minute>, with 24h time.

User: Invalid departure time. Use the format <day_of_week> <hour:minute>, with 24h time.

User: Invalid departure time. Use the format <day_of_week> <hour:minute>, with 24h time.

User: Invalid departure time. Use the format <day_of_week> <hour:minute>, with 24h time.

User: Invalid departure time. Use the format <day_of_week> <hour:minute>, with 24h time.

User: Invalid starting location.

User: Invalid ending location.

User: Invalid ending location.

User: Invalid positive integer capacity.

User: Invalid positive integer capacity.

User: Invalid positive integer capacity.

User: Source and destination cannot be the same place.

User: Flight 0
Departure:    Mon 10:05 Mumbai
Arrival:      Mon 11:40 NewDelhi
Distance:     1,139km
Duration:     1h 35m
Ticket Cost:  $317.79
Passengers:   0/80

User: Successfully added location Sydney.

User: Scheduling conflict! This flight clashes with Flight 0 departing from Mumbai on Monday 10:05.

User: Scheduling conflict! This flight clashes with Flight 0 departing from Mumbai on Monday 10:05.

User: Scheduling conflict! This flight clashes with Flight 0 departing from Mumbai on Monday 10:05.

User: Scheduling conflict! This flight clashes with Flight 0 arriving at NewDelhi on Monday 11:40.

User: Scheduling conflict! This flight clashes with Flight 0 arriving at NewDelhi on Monday 11:40.

User: Successfully added Flight 1.

User: Scheduling conflict! This flight clashes with Flight 1 departing from NewDelhi on Monday 12:40.

User: Flight 1
Departure:    Mon 12:40 NewDelhi
Arrival:      Mon 14:15 Mumbai
Distance:     1,139km
Duration:     1h 35m
Ticket Cost:  $365.43
Passengers:   0/120

User: Successfully added Flight 2.

User: Scheduling conflict! This flight clashes with Flight 2 departing from Sydney on Sunday 23:50.

User: Successfully added Flight 3.

User: Scheduling conflict! This flight clashes with Flight 3 departing from Sydney on Sunday 00:45.

User: Scheduling conflict! This flight clashes with Flight 3 departing from Sydney on Sunday 00:45.

User: Flights
-------------------------------------------------------
ID   Departure   Arrival     Source --> Destination
-------------------------------------------------------
   0 Mon 10:05   Mon 11:40   Mumbai --> NewDelhi
   1 Mon 12:40   Mon 14:15   NewDelhi --> Mumbai
   3 Sun 00:45   Sun 15:11   Sydney --> NewDelhi
   2 Sun 23:50   Mon 14:16   Sydney --> NewDelhi

User: Application closed.
